zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/NotebookRepoSync.java

```diff
@@ -39,6 +40,7 @@
 /**
  * Notebook repository sync with remote storage
  */
+@Immediate
```

Comment:
What is this annotation used for ?

Reply:
I think this can be removed, it's usually required at only one place. If you are doing it there ( `bindAsContract(NotebookRepoSync.class).to(NotebookRepo.class).in(Immediate.class);` ) then this becomes optional.
And hence changes in pom.xml can be reverted.